src/plugins/intel_cpu/tests/functional/single_layer_tests/convolution_backprop_data.cpp

```diff
@@ -200,7 +222,8 @@ class DeconvolutionLayerCPUTest : public testing::WithParamInterface<DeconvLayer
 
         std::tie(kernel, stride, padBegin, padEnd, dilation, convOutChannels, padType, outPadding) = basicParamsSet;
 
-        if (additionalConfig[InferenceEngine::PluginConfigParams::KEY_ENFORCE_BF16] == InferenceEngine::PluginConfigParams::YES) {
+        if (additionalConfig[InferenceEngine::PluginConfigParams::KEY_ENFORCE_BF16] ==
+            InferenceEngine::PluginConfigParams::YES) {
```

Comment:
If change this line to ov::hint::inference_precision, it will trigger a bug, so keep it here and create another separated PR to solve this bug.